pkgs/os-specific/linux/wireguard/0.0.20161209.nix

```diff
@@ -0,0 +1,9 @@
+{ stdenv, fetchurl, libmnl, iproute, kernel ? null }:
+
+# 0.0.20161216 introduced a non-backwards protocol change.
+# this version is kept around to have a version compatible with nixos stable.
```

Comment:
Judging by the fact that he closed the pull request, I believe this terrible idea is dead.